### PR TITLE
feat(PX-3910): add meta information for partner profile page

### PR DESCRIPTION
**Pull request by kiryl-zubarau**
This PR adds meta information for the partner profile page.

JIRA - https://artsyproduct.atlassian.net/browse/PX-3910

image

### CODE REVIEW

File: src/v2/Apps/Partner/Components/PartnerMeta/index.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+      <Meta property="profile:type" content={type} />
+      <Meta property="profile:username" content={slug} />
+
+      <Link rel="canonical" href={canonicalHref} />
+
+      {image && <Meta property="og:image" content={image} />}
+      {image && <Meta name="thumbnail" content={image} />}
+
+      {getENV("INCLUDE_ESCAPED_FRAGMENT") && (
```
Comment (dzucconi) [inline src/v2/Apps/Partner/Components/PartnerMeta/index.tsx:52]: Why this?

Comment (dzucconi) [reply src/v2/Apps/Partner/Components/PartnerMeta/index.tsx:52]: AFAIK this shouldn't be needed any longer

Comment (kiryl-zubarau) [reply src/v2/Apps/Partner/Components/PartnerMeta/index.tsx:52]: The previous version contains it, so I added it here too. Should I remove it?

Comment (dzucconi) [reply src/v2/Apps/Partner/Components/PartnerMeta/index.tsx:52]: Yeah is safe to remove now

File: src/v2/Apps/Partner/Components/PartnerMeta/index.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      <Meta property="og:title" content={title} />
+      <Meta property="og:description" content={description} />
+      <Meta property="og:url" content={href} />
+      <Meta property="og:type" content="profile" />
+
+      <Meta property="twitter:description" content={description} />
+      <Meta property="twitter:card" content="summary" />
+
+      <Meta property="profile:type" content={type} />
+      <Meta property="profile:username" content={slug} />
```
Comment (dzucconi) [inline src/v2/Apps/Partner/Components/PartnerMeta/index.tsx:45]: These profile tags should also be fine to remove. AFAIK they are open graph tags for social profile usernames? Weird

**Review by dzucconi — APPROVED** [review]
Feel free to merge when you're ready

**Pull request merged**